From 85ead3367ab86198a98190aa4869e59b1fc69c2f Mon Sep 17 00:00:00 2001
From: Anatol Pomozov <anatol.pomozov@gmail.com>
Date: Mon, 15 Feb 2016 17:49:22 -0800
Subject: [PATCH] a52: Make compatible with ffmpeg 3.0

audioconvert.h has been deprecated a while ago and got removed in
ffmpeg 3.0 (commit 2d40968dd3ff17b12f7).

Signed-off-by: Anatol Pomozov <anatol.pomozov@gmail.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 a52/pcm_a52.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/a52/pcm_a52.c b/a52/pcm_a52.c
index 88529eb..6fad77d 100644
--- a/a52/pcm_a52.c
+++ b/a52/pcm_a52.c
@@ -39,7 +39,7 @@
 #endif
 
 #if LIBAVCODEC_VERSION_INT >= AV_VERSION_INT(53, 34, 0)
-#include <libavutil/audioconvert.h>
+#include <libavutil/channel_layout.h>
 #include <libavutil/mem.h>
 #define USE_AVCODEC_FRAME
 #endif
-- 
1.7.11.7

